docs/build/isc/v1.3/docs/tutorials/defi-lend-borrow-tutorial-part-2.md

```diff
@@ -0,0 +1,89 @@
+# DeFi Lend Borrow UI
+
+## Part II 
+
```

Comment:
Can you mention that the tutorial is to be separated into a contracts part and UI part?

Reply:
Sure